mozci/push.py

```diff
@@ -1189,10 +1196,13 @@ def classify_regressions(
         else:
             all_groups = push_groups
 
+        groups_confirmed = {g.name: g.is_confirmed_failure() for g in all_groups}
+
         groups_consistent_failure = {
             g.name
             for g in all_groups
-            if g.is_consistent_failure(
+            if groups_confirmed[g.name] is True
+            or g.is_consistent_failure(
                 cross_config_counts[1] if cross_config_counts is not None else None,
                 consistent_failures_counts[1]
                 if consistent_failures_counts is not None
```

Comment:
With this approach, if a manifest is failing multiple times but the corresponding -cf tasks are not failing (so g.is_confirmed is False and g.is_consistent_failure is True) we still consider a group as a real failure. Should we instead do something like:
```g.is_confirmed is True or (g.is_confirmed is None and g.is_consistent_failure)```?

Reply:
yes, that makes sense;  this helps rule out the edge case where confirm-failure fails for reasons of test issues but the failure occurs in the scope of the manifest consistently.

This does bring up a different edge case, specifically:
 * task1: groupA: test1 fails
 * task2: groupA: test2 fails
 * task3: groupA: test3 fails
 * cf-task1: groupA: test1 passes
 
 this will look like groupA fails ALL the time but not in CF, so we would fail here.